### PR TITLE
companion commit for rstudio/rstudio-pro#2030

### PR DESCRIPTION
**Pull request by ricardofandrade**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -61,6 +61,7 @@
 
 ### RStudio Server Pro
 
+* **DEPRECATED:** The option `auth-proxy-require-hmac` is no longer operational and a warning will be issued to the logs if used. See the documentation on proxy authentication for alternatives. (Pro #2029)
```
Comment (jmcphers) [inline NEWS.md:64]: Nit: let's not lead with this as the very first thing people read about what we've done in this release! (put it towards the end of the notes)

**Review by jmcphers — CHANGES_REQUESTED** [review]

File: NEWS.md [hunk 2]
```diff
@@ -78,6 +78,7 @@
 * Changed the command to retrieve Slurm resource utilization to be run as the current user rather than the `slurm-service-user` (Pro #1527)
 * Reduced supurflous log messages in the Slurm Launcher Plugin log file about non-RStudio jobs in Slurm (Pro #1528)
 * Allow administrators to disabile the ability to set resource reqeusts on jobs launched through the Slurm Launcher Plugin (Pro #1948)
+* **DEPRECATED:** The option `auth-proxy-require-hmac` is no longer operational and a warning will be issued to the logs if used. See the documentation on proxy authentication for alternatives. (Pro #2029)
```
Comment (ricardofandrade) [inline NEWS.md:81]:
```suggestion
* **REMOVED:** The option `auth-proxy-require-hmac` is no longer operational and RStudio will not run if present. See the documentation on [Proxied Authentication] for alternatives to secure RStudio. (Pro #2029)
```

File: NEWS.md [hunk 3]
```diff
@@ -78,6 +78,7 @@
 * Changed the command to retrieve Slurm resource utilization to be run as the current user rather than the `slurm-service-user` (Pro #1527)
 * Reduced supurflous log messages in the Slurm Launcher Plugin log file about non-RStudio jobs in Slurm (Pro #1528)
 * Allow administrators to disabile the ability to set resource reqeusts on jobs launched through the Slurm Launcher Plugin (Pro #1948)
+* **REMOVED:** The option `auth-proxy-require-hmac` is no longer operational and RStudio will not run if present. See the documentation on [Proxied Authentication] for alternatives to secure RStudio. (Pro #2029)
```
Comment (ricardofandrade) [inline NEWS.md:81]:
```suggestion
* **REMOVED:** The option `auth-proxy-require-hmac` is no longer operational and RStudio will not run if present. See the documentation on [Proxy Security Considerations] for alternatives to secure RStudio. (Pro #2029)
```

File: NEWS.md [hunk 4]
```diff
@@ -78,6 +78,7 @@
 * Changed the command to retrieve Slurm resource utilization to be run as the current user rather than the `slurm-service-user` (Pro #1527)
 * Reduced supurflous log messages in the Slurm Launcher Plugin log file about non-RStudio jobs in Slurm (Pro #1528)
 * Allow administrators to disabile the ability to set resource reqeusts on jobs launched through the Slurm Launcher Plugin (Pro #1948)
+* **REMOVED:** The option `auth-proxy-require-hmac` is no longer operational and RStudio will not run if present. See the documentation on [Proxy Security Considerations] for alternatives to secure RStudio. (Pro #2029)
```
Comment (ricardofandrade) [inline NEWS.md:81]:
```suggestion
* **RETIRED:** The option `auth-proxy-require-hmac` is has been retired and it is no longer operational. RStudio will not start if enabled. See the documentation on [Proxy Security Considerations] for alternatives to secure RStudio. (Pro #2029)
```

**Pull request merged**